pkg/provisioner/controller.go

```diff
@@ -245,8 +246,8 @@ func (c *obcController) syncHandler(key string) error {
 	// Provision New Bucket or Grant Access to Existing Bucket
 	// *******************************************************
 	if !shouldProvision(obc) {
-		log.Info("skipping provision")
-		return nil
+		log.Info("updating OBC")
```

Comment:
comments on lines 238 and 246 should include that the event could also be update.